clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -55,7 +55,10 @@ async function initialize(context: vscode.ExtensionContext) {
   try {
     await vscode.workspace.fs.createDirectory(context.globalStorageUri);
   } catch (_error) {
-    throw Error(`Can't create global extension directory`);
+    outputChannel.appendLine(
```

Comment:
outputChannel is undefined at this point, also fix the message and extract it into a constant.

Reply:
should resolve : 83f21931e1b227c9067719b2eb37d9de80e20cfe